documentation/projects/proposals/rekognition_data/20240510-implementation_plan_machine_generated_tags_frontend.md

```diff
@@ -169,12 +168,18 @@ understand what the list is for. It is always desirable to have visible label
 next to a list of elements.
 
 The display of generated tags' provider and accuracy is, on the other hand, a
-challenging problem from accessibility standpoint. Tooltips are often described
-as inaccessible for mobile users who do not have a hover action. MDN's section
-on tooltip best practices suggests providing "clear labels and sufficient body
-text" instead of tooltips that hide important information. We could add the tag
-accuracy to the However, if we decide to add the provider to the generated tags,
-there will be a lot of duplication.
+challenging problem from accessibility standpoint. Tooltips proposed in the
+second alternative option are often described as inaccessible for mobile users
+who do not have a hover action. MDN's section on tooltip best practices suggests
+providing "clear labels and sufficient body text" instead of tooltips that hide
+important information.
+
+The third alternative option, where the provider is displayed once per the list
+of all the generated tags generated by it, and accuracy is displayed on the tag
+itself, is the most accessible option. The user will see (or hear) the provider
+and the accuracy of the tag without any additional actions. We need to make sure
+that the tag name and its accuracy are separated (such as with an invisible,
+`sr-only`, dot) to make the screen readers add a pause between them.
```

Comment:
Great call-out!

Reply:
Since we're deciding _not_ to go with this approach, do you mind changing the language (specifically `We need to make sure...`) to indicate that, _if we were to choose this approach down the line_, this is a consideration we'd need to take?